### ISSUE TITLE
HTML only custom elements should use binding context

### ISSUE THREAD

**Issue by Vheissu** [aurelia/aurelia#1085]
# 🐛 Bug Report

In Aurelia 1, when using a HTML only custom element (no view-model) the binding context view-model would be used. In Aurelia 2, HTML only custom elements create their own context and therefore,  if you need any values in your component, you need to create bindable properties and pass them in.

## 🤔 Expected Behavior

After speaking with @fkleuver I think we were in agreeance that custom elements which are HTML only should use the binding context. If I have a view model say `my-app.ts` and inside of `my-app.html` I reference a custom element called `user-list.html` which references an array called `users` which is loaded inside of `my-app.ts`.

## 😯 Current Behavior

Custom elements regardless of whether or not they are HTML only are run through the custom element implementation and a view-model is created for them.

## 🔦 Context

I want to create re-usable HTML only custom elements to break up my UI's, but for instances where I am merely displaying values and not worrying about loading additional data or anything else.

**Comment by bigopon** [comment]
I think we need a way to toggle this behavior at the very least.

Immediate thought for this is maybe it can be like this:
### Always create its own context, unless configured not so.

For example:
```html
<use-parent-scope></use-parent-scope>
```

The equivalent in script:
```ts
@customElement({
  useParentScope: true
})
```

Implication: needs to prevent/disallow a normal custom element to use this `useParentScope` option.

v1 templating primitives support this in a more natural way, because of the separation of view - view model. In v2, it's a bit different, which will require some work and refactoring.

**Comment by bigopon** [comment]
Resolved by #1184
